ephios/core/signals.py

```diff
@@ -128,6 +128,13 @@
 Receivers receive a ``request`` keyword argument. Receivers should return html that will be rendered inside a card.
 """
 
+register_group_permission_fields = PluginSignal()
+"""
+This signal is sent out to get a list of permission fields that should be displayed on the group form.
```

Comment:
is it really permission fields only we expect here? Could be any field (though it wouldn't do anything I guess?!)

Reply:
I remembered that we talked about adding PermissionFields via this signal because this is a common use case for plugins and keeps the interface simple. We argued that we can introduce something like the PluginForm on events and shifts if there is a need for pluigns to add custom forms. so i thought we agreed on only accepting permissionfields here

Reply:
Yeah thanks for explaining!